wallet-connect-test-bench/front-end/package.json

```diff
@@ -1,19 +1,21 @@
 {
   "name": "test-bench-for-wallets",
   "packageManager": "yarn@3.2.0",
-  "version": "1.5.3",
+  "version": "1.5.4",
   "license": "Apache-2.0",
   "type": "module",
   "engines": {
     "node": ">=16.x"
   },
   "dependencies": {
-    "@concordium/browser-wallet-api-helpers": "^2.4.0",
+    "@concordium/browser-wallet-api-helpers": "^3.0.0",
     "@concordium/react-components": "../../deps/concordium-dapp-libraries/packages/react-components",
     "@concordium/wallet-connectors": "../../deps/concordium-dapp-libraries/packages/wallet-connectors",
-    "@concordium/web-sdk": "^6.0.0",
+    "@concordium/web-sdk": "^7.0.0",
+    "@types/json-bigint": "^1.0.4",
     "@walletconnect/types": "^2.1.4",
     "eslint": "^8.37.0",
```

Comment:
`eslint` should also be in `devDependencies`.. It does not really make a difference as this is not a library intended for others to use, but would be good for good measure 😉